app/services/update_offer_conditions.rb

```diff
@@ -0,0 +1,28 @@
+class UpdateOfferConditions
+  def initialize(conditions:, application_choice:)
```

Comment:
Nitpicking: Can we reverse the order of the arguments, as we rely on the application choice in order to do anything, so it makes sens to have it as they primary argument.

```suggestion
  def initialize(application_choice:, conditions:)
```